R/conjunct_test_linter.R

```diff
@@ -7,13 +7,14 @@
 #'
 #' Similar reasoning applies to `&&` usage inside [stopifnot()] and `assertthat::assert_that()` calls.
 #'
-#' Relatedly, `dplyr::filter(DF, A & B)` is the same as `dplyr::filter(DF, A, B)`, but the
-#'   latter will be more readable / easier to format for long conditions. Note that this linter
-#'   assumes usages of `filter()` are `dplyr::filter()`; if you're using another function named `filter()`,
-#'   e.g. [stats::filter()], please namespace-qualify it to avoid false positives.
+#' Relatedly, `dplyr::filter(DF, A & B)` is the same as `dplyr::filter(DF, A, B)`, but the latter will be more readable
```

Comment:
looks like a lot of spurious diff here, any particular reason to widen the lines?

Reply:
Yes that's the upper limit... generally better to keep things narrower.

In this case, it's more of a distraction when reviewing to check line-by-line if there's any diff when the only "real" change is your added sentence. So, if you were writing documentation from scratch / a new paragraph, a whichever width you align to is fine. But here, adhering to the "prevaling" width makes review simpler by focusing the diff on what's actually changed.

(two paragraphs about a very minor point, alas)

Reply:
You should use a more powerful diff viewer/algorithm then :wink: I can very much recommend the Rust-based [diffr](https://github.com/mookid/diffr), which implements Myers' longest common subsequence algorithm.

Reply:
Tell it to the GitHub UI team :)